### DISCUSSION TITLE
I want to disable the display .grid class

### DISCUSSION THREAD

**Discussion by erolabzait** [tailwindlabs/tailwindcss#3068] [Help] [answered]
Is there a way to disable the display .grid class (or at least overwrite it with nothing), I use tailwind with a custom theme, and they use this class for other things (not related to display grid) and I don't want to use a prefix for tailwind (or overwrite their class with !important).

From what I read display is not available in theme.extend

**Comment by jonnitto** [comment]
Have you thought about it to use a prefix class for all tailwind classes?
https://tailwindcss.com/docs/configuration#prefix

**Comment by jonnitto** [comment, marked as answer]
Or you could disable the whole display classes like this:
```js
  module.exports = {
    corePlugins: {
     display: false,
    }
  }
```

Read more about this [here](https://tailwindcss.com/docs/display#disabling)

**Reply by erolabzait** [reply]
I tried that also, but I use the other visibility classes 🙃

**Reply by jonnitto** [reply]
You could use the  [`blocklist` option from PurgeCSS](https://purgecss.com/configuration.html#options) 

```js
module.exports = {
  purge: {
    content: ['./src/**/*.html'],

    // These options are passed through directly to PurgeCSS
    options: {
      blocklist: ['grid'],
    }
  },
}
```

**Reply by erolabzait** [reply]
blocklist worked. Thank you.
